### ISSUE TITLE
Add linter version to config file

### ISSUE THREAD

**Issue by maratori** [golangci/golangci-lint#563]
Here if feature request.

I'd like to fix version of `golangci-lint` in config file `.golangci.yml`.
It should fail if version of linter differs from config.
This is especially actual for local run (i.e. pre-commit). To be sure right version is used.

Example of config file:
```yml
version: v1.17.0

run:
  deadline: 2m

linters-settings:
  errcheck:
    check-type-assertions: true
  govet:
    check-shadowing: true
  goimports:
    local-prefixes: github.com/gtforge/atol_integration_service
  lll:
    line-length: 120
  unused:
    check-exported: true
  unparam:
    check-exported: true
  nakedret:
    max-func-lines: 30
  maligned:
    suggest-new: true

linters:
  disable-all: true
  enable:
    ## enabled by default
    - deadcode
    - errcheck
    - gosimple
    - govet
    - ineffassign
    - staticcheck
    - structcheck
    - typecheck
    - unused
    - varcheck
    ## disabled by default
    - bodyclose
    - depguard
    - dupl
    - goconst
    - gocritic
    - gocyclo
    - goimports
    - golint
    - gosec
    - interfacer
    - lll
    - maligned
    - misspell
    - nakedret
    - prealloc
    - scopelint
    - unconvert
    - unparam

issues:
  max-issues-per-linter: 0
  max-same-issues: 0
```
